### PR TITLE
[Jenkins] Move each test into its own step

### PR DESCRIPTION
**Pull request by DeepDiver1975**
Allows us to easier spot the failing test suite

@PVince81 @PhilippSchaffrath

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -14,83 +14,95 @@ timestampedNode('SLAVE') {
...
-		make test-php TEST_DATABASE=sqlite
-        	'''
-	}
-
-    stage 'PHPUnit'
+            sh '''
+            export NOCOVERAGE=1
+            unset USEDOCKER
+            phpenv local 7.1
+            make test-php TEST_DATABASE=sqlite
```
Comment (PVince81) [inline Jenkinsfile]: some tab vs space madness

**Review by PVince81 — COMMENTED** [review]
Hmm, okay, makes sense 👍

File: Jenkinsfile (hunk 1 above)
Comment (DeepDiver1975) [inline Jenkinsfile]: all spaces now in here .....

**Comment by PVince81** [comment]
@DeepDiver1975 can we backport this to stable9.1 and stable9 so we can also backport the other "blow up the test matrix" tests from https://github.com/owncloud/core/pull/26844 ?

**Comment by PVince81** [comment]
s/we/you/ 😉

**Comment by PVince81** [comment]
stable9.1: https://github.com/owncloud/core/pull/26976
stable9: TODO

**Pull request merged**